index.html

```diff
@@ -3,6 +3,7 @@
 <head>
     <meta charset="utf-8"/>
     <title>Renoma</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Abril+Fatface|Poppins">
```

Comment:
By security fonts, images and etc. should be downloaded in your project and used from that.